doc/classes/EditorResourcePreviewGenerator.xml

```diff
@@ -4,7 +4,7 @@
 		Custom generator of previews.
 	</brief_description>
 	<description>
-		Custom code to generate previews. Please check [code]file_dialog/thumbnail_size[/code] in [EditorSettings] to find out the right size to do previews at.
```

Comment:
Not only is this not a direct reference, but the mentioned property does not technically exist in the Editor Settings. Just an addendum.